build_tools/python/e2e_test_framework/models/tflite_models.py

```diff
@@ -129,3 +129,14 @@
     "https://storage.googleapis.com/iree-model-artifacts/efficientnet_lite0_int8_2.tflite",
     entry_function="main",
     input_types=["1x224x224x3xui8"])
+
+MOBILENET_V2_INT8 = common_definitions.Model(
+    name="MobileNetV2_int8",
```

Comment:
@pzread I would like to add the `id` field but I can't find how to define the ID if the model does not have a pre-exist run. Suggestions?

Reply:
What do you mean not having a pre-exist run?

You can add a new id to `unique_ids.py` following others' format. Postsubmit will upload the new ids to dashboard.

Reply:
Thanks. I add one line in build_tools/python/e2e_test_framework/models/README.md to explain how to generate the new uuid